metals/src/main/scala/scala/meta/internal/metals/BatchedFunction.scala

```diff
@@ -11,6 +11,15 @@ import scala.util.control.NonFatal
 
 import scala.meta.internal.async.ConcurrentQueue
 
+final case class BatchedCallbackData[A, B](
```

Comment:
Could we do the changes to BatchedFunction in another PR?

Reply:
Of course - https://github.com/scalameta/metals/pull/3571

Reply:
Thanks!